scripts/functions.sh

```diff
@@ -260,7 +260,7 @@ function geoserver_logging() {
 function file_env {
 	local var="$1"
 	local fileVar="${var}_FILE"
-	local def="${2:-}"
+	local def="${1:-}"
```

Comment:
Does this fix it for all env vars that can be called with `_FILE` ?

Reply:
Given the fact that this function takes just one argument in the [env-data.sh](https://github.com/kartoza/docker-geoserver/blob/master/scripts/env-data.sh) bash script, I expect to fix any env variable with **_FILE** suffix.